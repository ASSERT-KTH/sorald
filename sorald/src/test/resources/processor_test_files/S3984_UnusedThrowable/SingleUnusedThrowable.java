/*
Test case to repair a single unused throwable.
 */

public class SingleUnusedThrowable {
    public void unusedThrowable() {
        new IllegalArgumentException(); // Noncompliant
    }
}