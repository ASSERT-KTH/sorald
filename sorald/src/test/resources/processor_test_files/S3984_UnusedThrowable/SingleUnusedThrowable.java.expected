/*
Test case to repair a single unused throwable.
 */

public class SingleUnusedThrowable {
    public void unusedThrowable() {
        throw new IllegalArgumentException(); // Noncompliant
    }
}